TraceScripts/README.md

```diff
@@ -92,4 +92,27 @@ $> adb push <filepath>/PlayFabMultiplayerLogger.json /sdcard/PlayFabLogs/config/
     Please note that if you've modified the `logFolder` property of your `PlayFabMultiplayerLogger.json`, then you'll need to update the ADB command accordingly.
 ```
 $> adb pull /sdcard/PlayFabLogs/log/ <target local directory>
-```
\ No newline at end of file
+```
+
+### Linux: Instructions
+
+1. Download the [`PlayFabMultiplayerLogger.json` file](Linux/PlayFabMultiplayerLogger.json) from this repo.
+2. Open a new terminal.
+3. Create your `config` directory on the Linux device.
+```
+$> mkdir $HOME/PlayFabLogs/config/
+```
+4. You will need to modify the `logFolder` property of the `PlayFabMultiplayerLogger.json` file.
+```
+"logFolder": "/home/<your-username>/PlayFabLogs/log/"
+```
+- Please replace `<your-username>` with your username on your Linux machine. If you are unsure of your username:
+```
+$> echo $HOME
+```
+5. Copy the `PlayFabMultiplayerLogger.json` file into your `$HOME/PlayFabLogs/config/` directory.
+```
+$> cp <filepath>/PlayFabMultiplayerLogger.json $HOME/PlayFabLogs/config/
```

Comment:
Do you think displaying this command is necessary? Personally, finding it redundant but ok if you think it adds value.

Reply:
Prefer to over-inform rather than under-inform - plus it makes it easy for folks picking this up to just copy and paste.

Reply:
for the purposes of a second opinion, I also felt some of these instructions were a bit more verbose than I think we expect our target audience to need. 

for example, I'm not sure "open a terminal" needs to be a required step :) but also don't feel strongly enough that I'm happy deferring to your preference if you still prefer over-informing to underinforming.

Reply:
Yeah I also wasn't sure about that one on second read lol...I'll remove it.